app/components/framework/CompactAttributeInput/Matrix1dWidgetInput/index.tsx

```diff
@@ -96,10 +119,19 @@ function Row(props: RowProps) {
                     readOnly={readOnly}
                     disabled={disabled}
                 />
-            ) : (
-                <div className={styles.selectedValues}>
-                    {selectedValues}
-                </div>
+            )}
+            {!readOnly && suggestionModeEnabled && (
+                <MultiBadgeInput
+                    name={row?.key}
+                    onChange={handleCellsChange}
+                    options={sortedCells ?? []}
```

Comment:
MultiBadgeInput should support empty array. Let's not create new array on render function

Reply:
I've added a fixme